app/lib/languages/en.json

```diff
@@ -69,6 +69,10 @@
     "ledger": "Ledger",
     "ledgers": "Ledgers",
     "limit": "Limit",
+    "liquidity-pools": "Liquidity Pools",
+    "liquidity-pool.assets": "Pool Assets",
+    "liquidity-pool.shares": "Shares",
+    "liquidity-pool.trustlines": "Trustlines",
```

Comment:
[note]

I only added the English version.
Let me know ...
- whether the key names sound alright.
- if you want me to add translation for the other languages in this PR or later.

Reply:
> Although if you wanted to put the ja one in that would be better than a translation engine later

Added at https://github.com/chatch/stellarexplorer/pull/557/commits/7409ddea3cef7c36e621524aa6ec57b69850e3d6 .